### ISSUE TITLE
Interference between serialization and default pagination settings

### ISSUE THREAD

**Issue by postfalk** [bkg/django-spillway#21]

**Comment by postfalk** [comment]
The problem, I was chasing for days comes from the default pagination as defined in settings.py. If  change

```python
# settings.py

REST_FRAMEWORK = {
    # ...
    'PAGINATE_BY': 10,
    'PAGINATE_PY_PARAM': 'page_size',
    # ... 
}
```

to 

```python
REST_FRAMEWORK = {
    # ...
    'PAGINATE_BY': None,
    'PAGINATE_PY_PARAM': None,
    # ... 
}
```

things begun to work (without pagination obviously). Removing this keys entirely works as well and reflects the situation in your tests.

**Comment by bkg** [comment]
The default DRF settings do not enable pagination by default though they can be enabled as you have shown. See the recent example in #20.

Closing this as it's not clear from the description how this differs from #20, so let's keep the conversation going there.

**Comment by postfalk** [comment]
The difference seems to be between a bug and a feature. #21, I consider a
bug which could be solved by setting paginate_by and paginate_by_param on
the view in question. #20 is more concerning my wish that the GeoListView
should behave more like the DRF default and the GeoJsonRenderer should just
render whatever the view (including pagination) requests.

I know the original concern was to serve out a complete layer (which could
be achieved by turning pagination off) in the case described above.

I agree if #20 fully solved #21 would be unncessary.

On Fri, Jan 9, 2015 at 7:57 PM, Brian Galey <notifications@github.com>
wrote:

> Closed #21 <https://github.com/bkg/django-spillway/issues/21>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bkg/django-spillway/issues/21#event-216901760>.
>



-- 
Falk Schuetzenmeister, PhD
Web Application Developer
Geospatial Innovation Facility <http://gif.berkeley.edu/>

University of California Berkeley
111 Mulford Hall
94720 Berkeley, CA

schuetzenmeister@berkeley.edu
1-510-642-4897

**Comment by bkg** [comment]
Paginated and non-paginated responses are working as expected here and in line with the defaults provided by DRF. If you can provide reproducible steps to illustrate a bug, let's reopen it.
